tests/unit_tests/test_train_with_torch_scheduler.py

```diff
@@ -13,7 +13,7 @@ def update(self, *args, **kwargs) -> None:
         pass
 
     def compute(self):
-        return 1
```

Comment:
If you do float(value) in __maybe_get_item_from_tensor you won't need this patch

Reply:
Actually, I haven't found any place where the metric returns an integer value, so I decided to change it. It doesn't fail any tests and will be suitable when the issue with converting metric value to float is solved.